src/pages/iou/DistanceRequestPage.js

```diff
@@ -43,5 +68,8 @@ DistanceRequestPage.defaultProps = defaultProps;
 export default withOnyx({
     // We must provide a default value for transactionID here, otherwise the component won't mount
     // because withOnyx returns null until all the keys are defined
-    transactionID: {key: ONYXKEYS.IOU, selector: (iou) => (iou && iou.transactionID) || ''},
+    iou: {key: ONYXKEYS.IOU},
```

Comment:
NAB: We have a checklist item which states that you should use a selector to get only what you need. Please fix this in a follow up. Let's do everything in a follow up because this PR is huge already.

Reply:
`iou` is actually being used [here](https://github.com/Expensify/App/blob/3634718d0267e6cf45b24a628aca3097d25ee29a/src/pages/iou/NewDistanceRequestPage.js#L60), so I gonna keep the current code.